plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityCanBreakDoors.java

```diff
@@ -0,0 +1,42 @@
+package com.denizenscript.denizen.objects.properties.entity;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import org.bukkit.entity.Zombie;
+
+public class EntityCanBreakDoors extends EntityProperty<ElementTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name can_break_doors
+    // @input ElementTag(Boolean)
+    // @description
+    // If the entity is a Zombie, controls whether it can break doors.

```

Comment:
Should follow the general property meta format, E.g. `Whether a zombie can break doors` or something like that